recipes/x-transformers/recipe.yaml

```diff
@@ -0,0 +1,56 @@
+context:
+  version: "1.42.11"
+  python_min: 3.10
+
+package:
+  name: x-transformers
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/x/x-transformers/x_transformers-${{ version }}.tar.gz
+  sha256: eb342ffc5b86193354f27dcdbfd3e13746dd102bf5ddffa21f3286fdc07bde0e
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install . -vv --no-deps --no-build-isolation 
+
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python ${{ python_min }}
+    - setuptools
+    - wheel
+  run:
+    - python >=${{ python_min }}
+    - pytorch >=2.0.0
+    - einx >=0.3.0
+    - einops >=0.8.0
+    - packaging >=21.0.0
+
+tests:
+  - script:
+      - pytest -vvv tests
+    requirements:
+      run:
+        - pytest
+        - python >=${{ python_min }}
```

Comment:
For the tests this should be `- python ${{ python_min }}`, so that it tests the lowest allowable version of python. Can you also add an import test? See:
https://prefix-dev.github.io/rattler-build/latest/reference/recipe_file/#python-tests